src/commands/update-github-release.ts

```diff
@@ -0,0 +1,135 @@
+import * as Octokit from '@octokit/rest';
+import * as yargsParser from 'yargs-parser';
+import { getCurrentRepoNameWithOwner } from '../git/git';
+
+const BADGE = '[update-github-release]\t';
+
+type GitTag = string;
+type GitHubRepo = {
+  owner: string;
+  name: string;
+};
+
+export async function run(...args): Promise<boolean> {
+  const argv = yargsParser(args);
+
+  console.log(`${BADGE}`, argv);
+
+  const success = await updateGitHubRelease(argv.versionTag, argv.title, argv.text, !!argv.dry);
```

Comment:
Siehe anderer Kommentar. 

Würde hier sowas machen (unter der Annahme, dass `dry` einfach nur gegeben sein muss und keinen Wert hat):

```suggestion
  const isDryRun = argv.dry !== undefined;
  const success = await updateGitHubRelease(argv.versionTag, argv.title, argv.text, isDryRun);
```